package.json

```diff
@@ -36,9 +36,14 @@
     "build": "webpack --optimize-minimize --bail",
     "watch": "webpack --watch",
     "embed-assets": "node build/embed-assets.js",
-    "test": "npm run lint && npm run build && npm run api-check && npm run twbs-compat",
+    "test": "npm run lint && npm run build && npm run api-check && npm run twbs-compat && npm run ts-single && npm run karma-single",
     "twbs-compat": "webpack --env.twbs-compat --bail",
-    "semantic-release": "semantic-release pre && semantic-prerelease publish && semantic-release post"
+    "semantic-release": "semantic-release pre && semantic-prerelease publish && semantic-release post",
+    "watch-test": "npm run ts-watch & npm run karma-watch",
+    "ts-single": "./node_modules/.bin/tsc --project ./tests/",
+    "ts-watch": "./node_modules/.bin/tsc --watch --project ./tests/",
+    "karma-watch": "karma start --auto-watch --no-single-run",
+    "karma-single": "karma start --no-auto-watch --single-run --browsers PhantomJS"
```

Comment:
If I understand correctly, tests will run in PhantomJs as part of the theme verification, and with Chrome and PhantomJS locally. It seems that if we use only [Chrome headless](https://developers.google.com/web/updates/2017/06/headless-karma-mocha-chai) this may be simplified, and will require one less runner.